src/lambda/vue/samInvokeFrontend.ts

```diff
@@ -143,6 +146,19 @@ export default defineComponent({
             }
         })
 
+        Object.keys(this.$data).forEach(k => {
```

Comment:
Working with the proxy directly rather than manually assigning each state value means devs can add more things to the state without worrying about how they should save it. Previously, there seemed to be a misunderstanding with what was being saved (and how), so new fields were added to the model but never correctly stored.

This pattern is also easily transferable across different Vue webviews. The only edge-case is when this is applied with sub-components that share field names, but that is irrelevant here.